plugins/modules/helm_cli.py

```diff
@@ -0,0 +1,593 @@
+
+        fd, path = tempfile.mkstemp(suffix='.yml')
+        with open(path, 'w') as yaml_file:
+            yaml.dump(release_values, yaml_file, default_flow_style=False)
+        deploy_command += " -f=" + path
+
+    deploy_command += " --namespace=" + release_namespace
+    deploy_command += " " + release_name
+    deploy_command += " " + chart_name
+
+    return deploy_command
+
+
+# Delete release chart for helm2
+def delete_2(command, release_name, purge, disable_hook):
+    delete_command = command + " delete"
+
+    if purge:
+        delete_command += " --purge"
+
+    if disable_hook:
+        delete_command += " --no-hooks"
+
+    delete_command += " " + release_name
+
+    return delete_command
+
+
+# Delete release chart for helm3
+def delete_3(command, release_name, release_namespace, purge, disable_hook):
+    delete_command = command + " uninstall --namespace=" + release_namespace
+
+    if not purge:
+        delete_command += " --keep-history"
+
+    if disable_hook:
+        delete_command += " --no-hooks"
+
+    delete_command += " " + release_name
+
+    return delete_command
+
+
+def main():
+    global module, is_helm_2
+    module = AnsibleModule(
+        argument_spec=dict(
+            binary_path=dict(type='path'),
+            chart_ref=dict(type='path'),
+            chart_repo_url=dict(type='str'),
+            chart_repo_username=dict(type='str'),
+            chart_repo_password=dict(type='str', no_log=True),
+            chart_version=dict(type='str'),
+            release_name=dict(type='str', required=True, aliases=['name']),
+            release_namespace=dict(type='str', required=True, aliases=['namespace']),
+            release_state=dict(default='present', choices=['present', 'absent'], aliases=['state']),
+            release_values=dict(type='dict', default={}, aliases=['values']),
+            tiller_host=dict(type='str'),
+            tiller_namespace=dict(type='str', default='kube-system'),
+            update_repo_cache=dict(type='bool', default=False),
+
+            # Helm options
+            disable_hook=dict(type='bool', default=False),
+            force=dict(type='bool', default=False),
+            kube_context=dict(type='str'),
+            kubeconfig_path=dict(type='path', aliases=['kubeconfig']),
+            purge=dict(type='bool', default=True),
+            wait=dict(type='bool', default=False),
+            wait_timeout=dict(type='str'),
+        ),
+        required_if=[
+            ('release_state', 'present', ['release_name', 'chart_ref']),
+            ('release_state', 'absent', ['release_name'])
+
+        ],
+        required_together=[
+            ['chart_repo_username', 'chart_repo_password']
+        ],
+        supports_check_mode=True,
+    )
+
+    if not HAS_YAML:
+        module.fail_json(msg="Could not import the yaml python module. Please install `yaml` package.")
```

Comment:
Use `missing_required_lib` from basic.py.

Reply:
Fixed in last commit